package.json

```diff
@@ -6,10 +6,8 @@
     "build": "run-s build:*",
     "build:remix": "remix build",
     "build:server": "esbuild --platform=node --format=cjs ./server.ts --outdir=build --bundle",
```

Comment:
@pcattori Do we still need to build the server like this?
I guess we mainly did it to be used in the `dev` script 🤔

Reply:
still needed as this repo authors the custom server in TS and does not use `remix.config.js`'s `server` field to compile the custom server.

Reply:
Should we use `remix.config.js` instead?